addons/sale/models/sale_order.py

```diff
@@ -253,12 +253,14 @@ def _rec_names_search(self):
         comodel_name='payment.transaction',
         relation='sale_order_transaction_rel', column1='sale_order_id', column2='transaction_id',
         string="Transactions",
+        groups='account.group_account_user',
```

Comment:
Why not group_account_invoice (following the rights given in `account_payment` module) ?